apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/config/KeystoneHttpApiModule.java

```diff
@@ -91,17 +91,16 @@ protected void configure() {
       }
    }
 
+   // Allow providers to cleanly contribute their own aliases
```

Comment:
The [MapBinder](http://google.github.io/guice/api-docs/latest/javadoc/index.html?com/google/inject/multibindings/MapBinder.html) javadoc already has the detailed documentation of how it works and several examples. Replicating it here seems unnecessary. The object returned by this method already has the docs describing its usage.